flink-table/flink-table-planner/src/test/scala/org/apache/flink/table/api/TableEnvironmentTest.scala

```diff
@@ -656,14 +656,49 @@ class TableEnvironmentTest {
   def testExecuteSqlWithShowDatabases(): Unit = {
     val tableResult1 = tableEnv.executeSql("CREATE DATABASE db1 COMMENT 'db1_comment'")
     assertEquals(ResultKind.SUCCESS, tableResult1.getResultKind)
-    val tableResult2 = tableEnv.executeSql("SHOW DATABASES")
-    assertEquals(ResultKind.SUCCESS_WITH_CONTENT, tableResult2.getResultKind)
+    val tableResult2 = tableEnv.executeSql("CREATE DATABASE db2 COMMENT 'db2_comment'")
+    assertEquals(ResultKind.SUCCESS, tableResult2.getResultKind)
+    val tableResult3 = tableEnv.executeSql("CREATE DATABASE pre_db3 COMMENT 'db3_comment'")
+    assertEquals(ResultKind.SUCCESS, tableResult3.getResultKind)
+
+    val tableResult4 = tableEnv.executeSql("SHOW DATABASES")
+    assertEquals(ResultKind.SUCCESS_WITH_CONTENT, tableResult4.getResultKind)
     assertEquals(
       ResolvedSchema.of(Column.physical("database name", DataTypes.STRING())),
-      tableResult2.getResolvedSchema)
+      tableResult4.getResolvedSchema)
     checkData(
-      util.Arrays.asList(Row.of("default_database"), Row.of("db1")).iterator(),
-      tableResult2.collect())
+      util.Arrays.asList(Row.of("default_database"),
+        Row.of("db1"),
+        Row.of("db2"),
+        Row.of("pre_db3")).iterator(),
+      tableResult4.collect())
```

Comment:
Please use assertj builtin assertions for lists for that

Reply:
OK